pkg/util/azureclient/authz/remotepdp/types.go

```diff
@@ -32,8 +32,14 @@ type SubjectAttributes struct {
 	AltSecId         string   `json:"altsecid,omitempty"`
 	IdentityProvider string   `json:"idp,omitempty"`
 	Issuer           string   `json:"iss,omitempty"`
+	ClaimName        string   `json:"_claim_names,omitempty"`
 }
 
+// GroupExpansion is the value to be used with ClaimName in SubjectAttributes
+// This value gives CheckAccess a hint that it needs to retrieve all the groups the principal belongs to
+// and then give the response based on all group entitlements.
+const GroupExpansion = "{\"groups\":\"ENGORGIO~!\"}" // ref: https://en.wikibooks.org/wiki/Muggles%27_Guide_to_Harry_Potter/Magic/Engorgio
```

Comment:
I agree with Weinong. 

On top of that, it could be nice to use back [quotes](https://go.dev/ref/spec#String_literals) so we can put the raw string. 
eg:
```go
const GroupExpansion = `{"groups":"ENGORGIO~!"}` 
```
(we can use ` instead of ")
Maybe I am missing a good reason for not using it. 🙂